hail/python/test/hail/expr/test_expr.py

```diff
@@ -2858,6 +2858,7 @@ def test_set_numeric_functions(self):
 
     def test_uniroot(self):
```

Comment:
Also tests for an instance when you don't find a root, or that it still works when there's two possible roots, etc.

Reply:
I added tests for all of these except the one where you don't find a root; I feel like it's really hard to come up with a case where you'd expect that to happen, because of the constraint that min and max have to be on opposite sides of zero.

Reply:
True, makes sense. Thanks :+1: